auto-value-cursor/src/main/java/com/gabrielittner/auto/value/cursor/AutoValueCursorExtension.java

```diff
@@ -181,4 +201,30 @@ private FieldSpec createMapper(Context context) {
     private TypeName getFunc1TypeName(Context context) {
         return ParameterizedTypeName.get(FUNC1, CURSOR, getAutoValueClassClassName(context));
     }
+
+    private MethodSpec createToContentValuesMethod(ExecutableElement method,
+            Map<String, ExecutableElement> properties) {
+        MethodSpec.Builder builder = MethodSpec.methodBuilder(method.getSimpleName().toString())
+                .addModifiers(PUBLIC)
+                .returns(CONTENT_VALUES)
+                .addStatement("$T values = new $T($L)", CONTENT_VALUES, CONTENT_VALUES,
```

Comment:
You can do `$1T values = new $1T($2L)` so that you only have to pass `CONTENT_VALUES` once.